wled00/util.cpp

```diff
@@ -89,88 +89,43 @@ bool updateVal(const char* req, const char* key, byte* val, byte minv, byte maxv
 
 
 //append a numeric setting to string buffer
-void sappend(char stype, const char* key, int val)
+void sappend(Print& dest, char stype, const char* key, int val)
 {
-  char ds[] = "d.Sf.";
-
+  const __FlashStringHelper* type_str;
   switch(stype)
   {
     case 'c': //checkbox
-      oappend(ds);
-      oappend(key);
-      oappend(".checked=");
-      oappendi(val);
-      oappend(";");
+      type_str = F(".checked=");
```

Comment:
Why not use PSTR(...) and avoid casting?

Reply:
In an in-between version of the code, I had it set up so the print of that element was PROGMEM aware.  Given that this version just passes it in to printf as a '%s', you're correct that F() adds no extra value here -- instead we'll just do the alignment fault dance.